Inverted-Index/src/search/QueryParser.java

```diff
@@ -0,0 +1,34 @@
+package search;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import keywords.Keyword;
+import keywords.Ordinary;
+import keywords.Exclude;
+import keywords.Union;
+public class QueryParser {
+    private final String INCLUDE = "+";
```

Comment:
add these strings to a HashMap or some other convenient data structure and initialize it in the constructor.
it's more acceptable and readable, this code is good btw.